src/prefect/cli/flow_run.py

```diff
@@ -28,8 +28,8 @@
 )
 app.add_typer(flow_run_app, aliases=["flow-runs"])
 
-LOGS_PAGE_SIZE = 200
-LOGS_HEAD_DEFAULT_NUM_LINES = 20
+LOGS_DEFAULT_PAGE_SIZE = 200
+LOGS_DEFAULT_HEAD_NUM_LINES = 20
```

Comment:
This will be the constant that is used regardless of whether it is head, tail, or reverse, right? So I don't think having "head" in the name is needed. Maybe `LOGS_FLAG_DEFAULT_NUM_LOGS` or `LOGS_LIMIT_FLAG_DEFAULT_NUM_LOGS` or `LOGS_PREVIEW_FLAG_DEFAULT_NUM_LOGS`?

Reply:
LOGS_WITH_LIMIT_FLAG_DEFAULT_NUM_LOGS is indeed better